packages/xstate-react/test/useActorRef.test.tsx

```diff
@@ -611,10 +611,10 @@ describeEachReactMode('useActorRef (%s)', ({ suiteKey, render }) => {
     fireEvent.click(screen.getByText('Reload machine'));
 
     // while those numbers might be a little bit surprising at first glance they are actually correct
```

Comment:
the composition of functions changed between React versions. Previously it was smth like:
```ts
let children = renderWithHooks(...)
if (isSrictMode) {
  children = renderWithHooks(...)
}
```

where `renderWithHooks` had an extra loop based on `didScheduleRenderPhaseUpdateDuringThisPass`.

But now `renderWithHooks` itself is like
```ts
let children = callComponentInDEV(...);
if (didScheduleRenderPhaseUpdateDuringThisPass){
  children = renderWithHooksAgain(...)
}
if (isStrictMode) {
  children = renderWithHooksAgain(...)
}
```

Where `renderWithHooksAgain` has the infinite render loop protection